recreatingthepast-spring2020.md

```diff
@@ -0,0 +1,122 @@
+
+Students will be encouraged to begin a practice of generative visual sketching in whatever medium they feel comfortable in and share their process amongst the cohort. Class examples will be prepared in openFrameworks which is written in C++ and built on top of OpenGL. Although the assignments will be required to be completed via coding, the class is open to a variety of skill and experience levels. The lead teacher and teaching assistant will help to get you up to speed. Students are expected to have basic knowledge of coding and familiarity with using variables, functions, control structures. If you are a beginner, we recommend you take SFPC Coding Bootcamp before applying for this class. 
+
+### Syllabus 
+
+Here are examples of some of the artists / works / ideas we will be exploring:
+
+- Chaos and Order (Vera Molnar)
+- Animation and Harmony (John Whitney)
+- Computational Typography (Muriel Cooper / John Maeda)
+- Pattern (Bridgette Riley / Anni Albers)
+- Pixels (Ken Knowlton / Lillian F. Schwartz)
+- Images (Nancy Burson / Jason Salavon) 
+- Body (Camille Utterback and Romy Achituv / Myron Krueger)
+- Glitch / Deformation  (Rosa Menkman / Steina and Woody Vasulka)
+
+![](/static/img/recreating/recreatingthepast2.png) 
+
+### From our students
+
+"now, being months after taking this course, I think back on RTP often because the works we recreated are *so* relevant to the designs we see in our everyday walk down the street or strut through an exhibit. I look at fonts and typography and placement on posters. I analyze digital aesthetic structures with a more informed eye because I learned the how/what/why of appreciating the past. RTP gave me technical tools to learn how to appreciate the bedrock of the future - our past." Oceane Boulais (MIT) 
+
+"RTP is a great class with a strong workshop vibe. Using past works as a basis for creating new pieces is liberating, and it's gratifying to see all the pieces come together as an output of this journey! Zach's teaching style is refreshing as it blends discussions about different artistic perspectives with some practical coding tips to help us materialize our ideas." Joanne Leong (MIT)
+
+"Yes, Recreating The Past taught me a lot about coding. But for me, it’s value as an art history class, as an exercise in the importance of archiving, preservation, and curation, felt like a home I’d always been searching for. History is not a passive subject you simply consume, but an active form, something you “do,” and this was one of the only classes I’ve ever been in that seemed to not only understand that, but revel in it.” Max Neely Cohen (SFPC)
+
+"I loved it! It was the most balanced class in the course, with enough theory and history to make the practice more enjoyable.I particularly like the title of the class. It’s a way of saying something else. It could have been called “code”, but the name talks a lot about other things." Javier De Azkue (SFPC)
```

Comment:
```suggestion
"I loved it! It was the most balanced class in the course, with enough theory and history to make the practice more enjoyable. I particularly like the title of the class. It’s a way of saying something else. It could have been called “code”, but the name talks a lot about other things." Javier De Azkue (SFPC)
```